### PR TITLE
Add post submit for HNC

### PR DESCRIPTION
**Pull request by GinnyJI**
Test: none. I don't know how to test

### CODE REVIEW

File: config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-postsubmit.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      testgrid-dashboards: wg-multi-tenancy-hnc
+      testgrid-tab-name: postsubmit-tests
+    decorate: true
+    path_alias: sigs.k8s.io/multi-tenancy
+    run_if_changed: "incubator/hnc/.*"
+    spec:
+      containers:
+      - image: golang:1.14
+        command:
+        - ./incubator/hnc/hack/ci-test.sh
```
Comment (adrianludwin) [inline config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-postsubmit.yaml:14]: We should fork this file before merging this PR so we don't need to modify `test-infra` when our presubmits and postsubmits diverge (which they should very soon).

/hold

Comment (GinnyJI) [reply config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-postsubmit.yaml]: done

Comment (adrianludwin) [reply config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-postsubmit.yaml]: Well, done once https://github.com/kubernetes-sigs/multi-tenancy/pull/1080 is merged, right? :)

**Comment by GinnyJI** [comment]
Looking for a reliable way to test the post submit script, close this PR for now.
/close

**Pull request closed without merging**